browser-test/src/admin_program_creation.test.ts

```diff
@@ -1,64 +1,39 @@
-    await page.fill('text=Question help text', 'helpText');
+    const adminPrograms = new AdminPrograms(page);
 
-    // Add three options
-    await page.click('#add-new-option');
-    await page.fill('input:above(#add-new-option)', 'chocolate');
-    await page.click('#add-new-option');
-    await page.fill('input:above(#add-new-option)', 'vanilla');
-    await page.click('#add-new-option');
-    await page.fill('input:above(#add-new-option)', 'strawberry');
+    await adminQuestions.addAddressQuestion('apc-address');
+    await adminQuestions.addNameQuestion('apc-name');
+    await adminQuestions.addTextQuestion('apc-text');
+    await adminQuestions.addRepeaterQuestion('apc-repeater');
 
-    // Assert there are three options present
-    var questionSettingsDiv = await page.innerHTML('#question-settings');
-    expect(questionSettingsDiv.match(/<input/g)).toHaveLength(3);
 
-    // Remove first option - use :visible to not select the hidden template
-    await page.click('button:text("Remove"):visible')
+    const programName = 'apc program';
+    await adminPrograms.addProgram(programName);
+    await adminPrograms.editProgramBlock(programName, 'apc program description');
 
-    // Assert there are only two options now
-    questionSettingsDiv = await page.innerHTML('#question-settings');
-    expect(questionSettingsDiv.match(/<input/g)).toHaveLength(2);
+    // All questions should be available in the question bank
+    expect(await page.innerText('id=question-bank-questions')).toContain('apc-address');
+    expect(await page.innerText('id=question-bank-questions')).toContain('apc-name');
+    expect(await page.innerText('id=question-bank-questions')).toContain('apc-text');
+    expect(await page.innerText('id=question-bank-questions')).toContain('apc-repeater');
 
-    // Verify question preview text has changed based on user input.
-    expect(await page.innerText('.cf-applicant-question-text'))
-        .toContain("questionText");
-    expect(await page.innerText('.cf-applicant-question-help-text'))
-        .toContain("helpText");
+    // Add a non-repeater question and the repeater option should go away
+    await page.click('button:text("apc-name")');
+    expect(await page.innerText('id=question-bank-questions')).not.toContain('apc-repeater');
 
-    // Submit the form, then edit that question again
-    await page.click('text=Create');
-    await adminQuestions.expectDraftQuestionExist(questionName);
-
-    // Edit the question
-    await adminQuestions.gotoQuestionEditPage(questionName);
-    var questionSettingsDiv = await page.innerHTML('#question-settings');
-    expect(questionSettingsDiv.match(/<input/g)).toHaveLength(2);
+    // Remove the non-repeater question and add a repeater question. All options should go away..
+    await page.click('button:text("apc-name")');
+    await page.click('button:text("apc-repeater")');
+    expect(await page.innerText('id=question-bank-questions')).toBe('Question bank');

```

Comment:
Suggestion (if-minor): could we test repeated questions in the repeater?

Reply:
I need some more clarification on this.

If you mean checking that the repeated questions are available in repeated blocks, there's no way to create repeated blocks yet. That's next!

If you mean repeated questions in the repeater block, that isn't allowed. A repeater block can only have one question and it has to be a repeater question.

Reply:
Yes I meant repeated questions in repeated blocks. Looking forward to it!